website/recipients/admin.py

```diff
@@ -131,9 +131,10 @@ class MealRequestAdmin(admin.ModelAdmin):
         'notify_deliverers_details',
     )
     search_fields = (
+        '=id',
         'name',
-        'email',
-        'phone_number',
+        '=email',
+        '=phone_number',
```

Comment:
```suggestion
        'email',
        'phone_number',
```

Let's keep these ones as generically searchable, often an organizer will want to search by just one part of the phonenumber/email.